site/docs/composition/composition.md

```diff
@@ -14,7 +14,7 @@ Vega-Lite's compiler infers how input data should be reused across constituent v
 ## Documentation Overview
 {:.no_toc}
 
-* TOC
+- TOC
```

Comment:
Not sure if  `- TOC` works for the special command for Table of content.  
Have you run the website locally to check if this works?

Reply:
![image](https://user-images.githubusercontent.com/21237466/36938328-fb5c6990-1f45-11e8-9f9c-3ed04080bcc3.png)
It's working.